toolchains/unittest/BUILD

```diff
@@ -0,0 +1,51 @@
+load("//lib:unittest.bzl", "TOOLCHAIN_TYPE", "unittest_toolchain")
+
+toolchain_type(
+    name = "toolchain_type",
+)
+
+unittest_toolchain(
+    name = "exec_batch",
+    failure_templ = "\n".join([
```

Comment:
Is there a reason not to use a multiline string instead of join? If so, add a comment explaining why.

Reply:
I think it does not improve readability as much as it screws up indentation. I did it nonetheless, partly to see how it looks and partly because I have no string preference for either style.